File: LeetCode/Easy/1791-find-center-of-star-graph/1791-find-center-of-star-graph.cpp
class Solution {
public:
    int findCenter(vector<vector<int>>& edges) {
        int n = edges.size();
        vector<bool> count(n+2);
        
        for (vector<int>& v : edges){
            if (count[v[0]]) return v[0];
            if (count[v[1]]) return v[1];
            count[v[0]] = true;
            count[v[1]] = true;
        }

        return -1;
    }
};

File: LeetCode/Easy/1791-find-center-of-star-graph/1791-find-center-of-star-graph.py
class Solution:
    def findCenter(self, edges: List[List[int]]) -> int:
        count = [0] * (len(edges) + 2)
        ret = 0
        
        for v0, v1 in edges:
            count[v0] += 1
            count[v1] += 1
        
        for i in range(1, len(count)):
            if count[i] > ret:
                ret = i
        
        return ret